README.md

```diff
@@ -31,31 +31,27 @@ After adding the snapshots repository, you can add the `google-cloud-graalvm-sup
 
 This dependency contains the GraalVM configurations (reflection config, native call config) to provide out-of-the-box support for native-image compilation of applications depending on the Google Java Client Libraries.
 
-### Note on grpc-netty-shaded version
+### Client Library Versions
 
-As an additional step, you will also need to override the version of `grpc-netty-shaded` to version `1.32.1` (or later):
+To get your application working with GraalVM compilation, you will also have to ensure that the client library versions in your project are supported by your chosen version of `google-cloud-graalvm-support`.
```

Comment:
"To compile with GraalVM (`native-image`?), ensure the client library version in your project is supported by `google-cloud-graalvm-support`"

Reply:
Done.